doc/conf.py

```diff
@@ -299,6 +299,7 @@
     "auto_examples/decomposition/plot_beta_divergence": (
         "auto_examples/applications/plot_topics_extraction_with_nmf_lda"
     ),
+    "auto_examples/svm/plot_svm_nonlinear": "auto_examples/svm/plot_svm_kernels",
```

Comment:
As noted in https://github.com/scikit-learn/scikit-learn/pull/26972#discussion_r1291838864, the XOR problem is the canonical example for non-linear decision functions.

I agree that `auto_examples/svm/plot_svm_kernels` illustrates similar motivations but I would rather redirect to an example that also includes the canonical XOR, maybe with more models / kernels as suggested by Guillaume.

Reply:
@ogrisel  Do you think these days this example is still relevant? I would rather people read more realistic examples instead. Is there another example you prefer to be here instead?

I also don't really see a way where users would actually benefit from this redirection. Our example pages don't have a link to the same example in the new release. The only link we provide is to the home page of the latest release, not the same page in the latest release. So I'd be happy to remove these redirections altogether.

Reply:
I was thinking about adding XOR to classifier comparison, wdyt?

Reply:
I would be inclined to have this dataset because this is still an interesting synthetic one. Probably the classification one is the right place to have it. For instance, this is part of the default dataset in this playground (https://playground.tensorflow.org)

Reply:
So let's remove this one, and put the dataset into an existing one. We really have too many examples!